google/cloud/spanner_dbapi/parse_utils.py

```diff
@@ -19,6 +19,117 @@
 from .types import DateStr, TimestampStr
 from .utils import sanitize_literals_for_upload
 
+TYPES_MAP = {
+    bool: spanner.param_types.BOOL,
+    bytes: spanner.param_types.BYTES,
+    str: spanner.param_types.STRING,
+    int: spanner.param_types.INT64,
+    float: spanner.param_types.FLOAT64,
+    datetime.datetime: spanner.param_types.TIMESTAMP,
+    datetime.date: spanner.param_types.DATE,
+    DateStr: spanner.param_types.DATE,
+    TimestampStr: spanner.param_types.TIMESTAMP,
+}
```

Comment:
I expect we will be in need of these cast map often, so moving it to a constant.